_maps/configs/inteq_talos.json

```diff
@@ -28,15 +28,15 @@
 		},
 		"Corpsman": {
 			"outfit": "/datum/outfit/job/paramedic/inteq",
-			"slots": 1
+			"slots": 2
 		},
 		"Enforcer": {
 			"outfit": "/datum/outfit/job/security/inteq",
 			"slots": 1
 		},
 		"Recruit": {
 			"outfit": "/datum/outfit/job/assistant/inteq",
-			"slots": 7
+			"slots": 4
```

Comment:
I would honestly consider adding another enforcer and knocking this down to 3 or even 2.

Reply:
Hard no on the second enforcer- It's a support ship, after all, and I want to keep the combat crew relatively scarce. We'll see how the recruit cap plays out in testing.